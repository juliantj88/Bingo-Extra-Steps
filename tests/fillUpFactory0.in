1
a
b
1 Y 1
2 U 1
3 B 2
4 L 2
5 U 3
save fillUpFactory0
4