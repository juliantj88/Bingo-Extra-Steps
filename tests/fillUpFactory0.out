a
b
0
0
0
FRRRBBYYYYUUURRL
....
....
....
....
....
byrul
lbyru
ulbyr
rulby
yrulb
byrul
lbyru
ulbyr
rulby
yrulb
Y
B.
U..
....
.....
U
L.
...
....
.....
       
       

RRLBBYRLLUYBURRBLRYRBLBUUBRLBLULYRLYLLUYYULLLRLUYBLBYYYRBUURUBBYUBYRURBYYUBBURLL
0